script.js

```diff
@@ -0,0 +1,123 @@
+    desc: 'Benkler studies commons-based peer production, and published his seminal book, The Wealth of Networks in 2006',
+    image: './images/speaker_3.png',
+  },
+  {
+    id: 4,
+    name: 'Yochai Benkler',
+    title:
+      'Berkman Professor of Entrepreneurial Legal Studies at Harvard Law School',
+    desc: 'Benkler studies commons-based peer production, and published his seminal book, The Wealth of Networks in 2006',
+    image: './images/speaker_4.png',
+  },
+  {
+    id: 5,
+    name: 'Yochai Benkler',
+    title:
+      'Berkman Professor of Entrepreneurial Legal Studies at Harvard Law School',
+    desc: 'Benkler studies commons-based peer production, and published his seminal book, The Wealth of Networks in 2006',
+    image: './images/speaker_5.png',
+  },
+  {
+    id: 6,
+    name: 'Yochai Benkler',
+    title:
+      'Berkman Professor of Entrepreneurial Legal Studies at Harvard Law School',
+    desc: 'Benkler studies commons-based peer production, and published his seminal book, The Wealth of Networks in 2006',
+    image: './images/speaker_6.png',
+  },
+];
+
+const loadMoreBtn = document.getElementById('load-more-btn');
+const speakerCards = document.getElementById('speaker-cards');
+let out = '';
+
+const loadItem = 2;
+const screenSize = window.innerWidth;
+window.addEventListener('load', () => {
+  for (let i = 0; i < dataObj.length; i += 1) {
+    const element = dataObj[i];
+    if (screenSize < 768 && i < loadItem) {
+      out += `
+      <div class="speaker-card">
+        <img src="${element.image}" alt="icon">
+        <div class="speaker-card-content">
+            <h4>${element.name}</h4>
+            <p>${element.title}</p>
+            <hr>
+            <small>${element.desc}</small>
+        </div>
+      </div>
+      `;
+    } else if (screenSize >= 768) {
+      out += `
+      <div class="speaker-card">
+      <img src="${element.image}" alt="icon">
+      <div class="speaker-card-content">
+            <h4>${element.name}</h4>
+            <p>${element.title}</p>
+            <hr>
+            <small>${element.desc}</small>
+        </div>
+      </div>
+      `;
+    }
+  }
+  const div = document.createElement('div');
+  div.classList.add('speaker-grid');
+  speakerCards.insertBefore(div, loadMoreBtn);
+  div.innerHTML = out;
+
+  loadMoreBtn.addEventListener('click', () => {
+    for (let i = 0; i < dataObj.length - 2; i += 1) {
+      const element = dataObj[i];
+      out += `
+      <div class="speaker-card">
+      <img src="${element.image}" alt="icon">
+      <div class="speaker-card-content">
+            <h4>${element.name}</h4>
+            <p>${element.title}</p>
+            <hr>
+            <small>${element.desc}</small>
+        </div>
+      </div>
+      `;
+    }
+    // let div = document.createElement("div");
+    // div.classList.add("speaker-grid");
+    // speakerCards.insertBefore(div, loadMoreBtn);
```

Comment:
- [ ] Kindly remove this commented code in order to make your project clean according to the best practises. 👍